src/Kodify/BlogBundle/Resources/views/Post/view.html.twig

```diff
@@ -9,5 +9,34 @@
         <div class="panel-body">
             {{ post.content }}
         </div>
+        <div class="panel-footer">
+            {% include 'KodifyBlogBundle:Post:Component/rating.html.twig' %}
+        </div>
     </div>
-{% endblock %}
\ No newline at end of file
+
+    <section id="comments">
+        <header>
+            <h1>Comments</h1>
+        </header>
+        <ul class="list-unstyled">
+            {% for comment in post.comments %}
+                <li class="comment">
+                    <div>
+                        <span class="comment-author">{{ comment.author.name }}</span>
+                    </div>
+                    <div>
+                        <span class="comment-text">{{ comment.text }}</span>
+                    </div>
+                    <div class="comment-date">
+                        {{ comment.createdAt|date('d/m/Y H:i') }}
+                    </div>
+                </li>
+            {% else %}
+                <li class="no-comments">There are no comments</li>
+            {% endfor %}
```

Comment:
Hi,
We have at least 2 options, we can make an AJAX request to a controller that returns a full html view or a JSON(like an API), an API, or refresh the full page passing the filters, order, limit... in the query. Of course we can't retrieve post directly by lazy loading from post object. In one of the last  commits this is refactored to dont do it, and comments are passed from controlled already "filtered".